docs/docs/Configuration/configuration-cli.mdx

```diff
@@ -72,88 +87,167 @@ The following sections describe the available CLI commands and any non-default o
-# or
-python -m langflow copy-db
 ```
 
-The database files are `langflow.db` and `langflow-pre.db`.
-If these files don't exist in the cache directory, then nothing is copied.
+  </TabItem>
+</Tabs>
 
 ### langflow migration
 
-Run or test database migrations:
+Manages database schema migrations for Langflow using [Alembic](https://alembic.sqlalchemy.org/en/latest/), a database migration tool for SQLAlchemy.
+
+The `migration` command has two modes:
+
+* Test mode checks if migrations can be applied safely without actually running the migrations.
+This is the default mode, and should be run before running `--fix`.
+
+* Fix mode actually applies the migrations to update your database schema.
+This is a destructive operation that can potentially delete data, so run `langflow migration --test` before running `--fix`.
+
+<Tabs groupId="Invocation">
+  <TabItem value="uv (recommended)" label="uv (recommended)" default>
+
+```bash
+uv run langflow migration [OPTIONS]
+```
+
+  </TabItem>
+  <TabItem value="Global" label="Global">
 
 ```bash
 langflow migration [OPTIONS]
-# or
-python -m langflow migration [OPTIONS]
 ```
 
+  </TabItem>
+</Tabs>
+
 #### Options
 
 | Option | Default | Type | Description |
 |--------|---------|--------|-------------|
-| `--test` | `true` | Boolean | Run migrations in test mode. Use `--no-test` to disable test mode. |
-| `--fix` | `false` (`--no-fix`) | Boolean | Fix migrations. This is a destructive operation, and all affected data will be deleted. Only use this option if you know what you are doing. |
+| `--test` | `test` | Boolean | Run migrations in test mode. Use `--no-test` to disable test mode. |
+| `--fix` | `no-fix` | Boolean | Fix migrations. This is a destructive operation, and all affected data will be deleted.|
 
 ### langflow run
 
 Start the Langflow server.
 
+<Tabs groupId="Invocation">
+  <TabItem value="uv (recommended)" label="uv (recommended)" default>
+
+```bash
+uv run langflow run [OPTIONS]
+```
+
+  </TabItem>
+  <TabItem value="Direct" label="Direct">
+
 ```bash
 langflow run [OPTIONS]
-# or
-python -m langflow run [OPTIONS]
 ```
 
+  </TabItem>
+</Tabs>
+
 #### Options
 
 | Option | Default | Type | Description |
 |--------|---------|--------|-------------|
-| <Link id="run-host"/>`--host` | `localhost` | String | The host on which the Langflow server will run. |
-| <Link id="run-workers"/>`--workers` | `1` | Integer | Number of worker processes. |
-| <Link id="run-worker-timeout"/>`--worker-timeout` | `300` | Integer | Worker timeout in seconds. |
-| <Link id="run-port"/>`--port` | `7860` | Integer | The port on which the Langflow server will run. The server automatically selects a free port if the specified port is in use. |
-| <Link id="run-components-path"/>`--components-path` | `langflow/components` | String | Path to the directory containing custom components. |
+| `--auto-saving` | `true` | Boolean | Enable flow auto-saving. Use `--no-auto-saving` to disable flow auto-saving. |
```

Comment:
For the booleans, I think it's a bit misleading to say the "default" is true/false when you don't pass `--option false` or `--option true`.

IMO the "real" default is `--option` or `--no-option`. I think it would be more helpful for the Default column to use one of the following styles for Booleans:

```suggestion
| `--auto-saving` | True  | Boolean | Enable flow auto-saving. Use `--no-auto-saving` to disable flow auto-saving. |
```

```suggestion
| `--auto-saving` | `--auto-saving` (true)  | Boolean | Enable flow auto-saving. Use `--no-auto-saving` to disable flow auto-saving. |
```